src/MSBuild/XMake.cs

```diff
@@ -545,7 +545,7 @@ string commandLine
 #endif
                 int cpuCount = 1;
 #if FEATURE_NODE_REUSE
-                bool enableNodeReuse = NativeMethodsShared.IsWindows;
```

Comment:
nit: Shame this has to be repeated twice. It would be nice if BuildParameters had a Default singleton to access default values. But then it isn't immutable, so the Default can get mutated :(

Reply:
Yeah, I felt the same way when I was making the original temporary change I'm undoing here. I think some of them could have been avoided but was making a minimum change.